### ISSUE TITLE
Ignore project

### ISSUE THREAD

**Issue by abisaicruz-delbravo** [zubroide/gitpab#48]
Possibility of ignoring projects in the import

**Comment by mnvx** [comment]
I don't understand the problem. Need details.

**Comment by adelbravo** [comment]
the problem is that I want to ignore a single gitlab project and import all others, I do not see the option to ignore projects only the option to include one by one the projects that I want to import and this option would have to be modified every time I add a new one project

**Comment by mnvx** [comment]
Yes, currently there is no option to choose namespace for import.
I would to do it, it is really would helpful for my projects too. But currently I have not time for it. If anyone may do it - welcome.
Also will be great to let load projets from different Gitlab hosts into one Gitpab instance.
